code/modules/food_and_drinks/food.dm

```diff
@@ -19,6 +19,9 @@
 	if(last_check_time + 50 < world.time)
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
+			if((H.skin_tone == "caucasian1" || H.skin_tone == "albino") && prob(10))
+				to_chat(H,"<span class='warning'>Whew, that food was spicy!</span>")
+				H.reagents.add_reagent("capsaicin", 3)
```

Comment:
racism

Reply:
you can't be racist towards whites check your privilege

Reply:
I'm Texan and this offends my spicy tolerance